src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -537,3 +537,37 @@ export const getAttachmentsSchema = defaultAttachmentId => {
     attachments,
   );
 };
+
+const isDateRange = ({ from, to }) => !!(from && to);
+
+const parseDate = dateString => moment(dateString, 'YYYY-MM-DD');
+
+// NOTE: Could move this to outside all-claims
+/**
+ * Checks to see if the first parameter is inside the date range (second parameter).
+ * If the first parameter is a date range, it'll return true if both dates are inside the range.
+ * @typedef {Object} DateRange
+ * @property {string} to - A date string YYYY-MM-DD
+ * @property {string} from - A date string YYYY-MM-DD
+ * ---
+ * @param {String|DateRange} inside - The date or date range to check
+ * @param {DateRange} outside - The range `inside` must fit in
+ * @param {String} inclusivity - See https://momentjs.com/docs/#/query/is-between/
+ *                               NOTE: This function defaults to inclusive dates which is different
+ *                               from moment's default
+ */
+export const isWithinRange = (inside, outside, inclusivity = '[]') => {
```

Comment:
had a little bit of trouble parsing this based on the param names, but should've probably read the jsdoc 😛

Reply:
Yeahhhhhh I don't like the naming. Any suggestions?